tests/test_learnerid.py

```diff
@@ -1,60 +0,0 @@
-from contextlib import nullcontext as does_not_raise
```

Comment:
we might consider moving these tests to `test_strategy.py` instead of deleting altogether

Reply:
That sounds good! I was too focus on making the tests pass :)